tensorflow/contrib/layers/python/layers/layers.py

```diff
@@ -1831,7 +1831,7 @@ def separable_convolution2d(
     padding='SAME',
     rate=1,
     activation_fn=nn.relu,
-    normalizer_fn=None,
+    normalizer_fn=batch_norm,
```

Comment:
This is meant to be None. Every other layer (in particular conv layers) defaults to None.

Such a change would also break many models.